docker/linux/android/Dockerfile

```diff
@@ -18,10 +18,11 @@ FROM ${FROM_IMAGE:-cobalt-build-base}
 # TODO: b/309157124 - add --no-install-recommends.
 RUN apt update -qqy
 RUN apt -y install wget curl
-RUN wget https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.deb
+# Jdk17, verify SHA256 of the installation file, then install it.
+RUN h=$(mktemp); t=./jdk-17_linux-x64_bin.deb; echo '0871ec2276b3440ab03e4bb1e77a9d7645f82eb9d6c449d743a585845c370269' > $h; wget 'https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.deb' -qO "$t" && if sha256sum "$t" |grep -Eo '^\w+' |cmp -s $h ; then apt install -qqy $t ; else echo ERROR: Invalid SHA256 hash ; fi ; rm "$t"; rm "$h";
```

Comment:
This `RUN` directive should be moved down, below the `apt` step.

Reply:
done